config/logger.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const winston = require('winston')
+
+module.exports.setupLogger = function () {
+  return winston.createLogger({
```

Comment:
This could take advantage of the singleton pattern with Node modules where the module is initialised once and then shared by all other modules -- what is happening here is a new instance (and object to track/ store in memory) is being created by every file that requires logging. 

Suggestion -- 

```js
const winston = require('winston')

const config = {} //...
const logger = winston.createLogger(config)

module.exports = logger
```

```js 
const logger = require('../../config/logger')
logger.info()
```

In this scenario the (relatively large) instance is only created once and is then used by all modules.

Reply:
I agree. I'd much rather introduce a shared context of some sort, but decided to go the way it use to be in this project 🤔 

I'll make that change if you're happy with it. 🎉